backend/src/main/java/com/sscatalog/specialistsservicescatalog/services/ServiceRequestsService.java

```diff
@@ -0,0 +1,34 @@
+package com.sscatalog.specialistsservicescatalog.services;
+
+import com.sscatalog.specialistsservicescatalog.dtos.MakeServiceRequestRequest;
+import com.sscatalog.specialistsservicescatalog.entities.*;
+import com.sscatalog.specialistsservicescatalog.exceptions.ApiException;
+import com.sscatalog.specialistsservicescatalog.repositories.OfferedServiceRepository;
+import com.sscatalog.specialistsservicescatalog.repositories.ServiceRequestRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ServiceRequestsService {
+
+    private final OfferedServiceRepository offeredServiceRepository;
+
+    private final ServiceRequestRepository serviceRequestRepository;
+
+    public ServiceRequestsService(OfferedServiceRepository offeredServiceRepository,
+                                  ServiceRequestRepository serviceRequestRepository) {
+        this.offeredServiceRepository = offeredServiceRepository;
+        this.serviceRequestRepository = serviceRequestRepository;
+    }
+
+    public void makeServiceRequest(User user, MakeServiceRequestRequest request) {
+        OfferedService requestedService = offeredServiceRepository.findById(request.getRequestedServiceId())
+                                                                  .orElseThrow(() -> new ApiException(
+                                                                          "Offered service does not exists"));
+        Specialist requestedServiceSpecialist = requestedService.getSpecialist();
+        if (requestedServiceSpecialist.getUser().equals(user)) {
```

Comment:
Better practice is to use Objects.equals(requestedServiceSpecialist.getUser(), user) since you will not get NullPointerException if requestedServiceSpecialist.getUser() returns null for some reason.